apps/omg_utils/lib/omg_utils/http_rpc/response.ex

```diff
@@ -72,7 +72,7 @@ defmodule OMG.Utils.HttpRPC.Response do
 
   defp to_response(data, result),
     do: %{
-      version: "1.0",
+      version: "0.2",
```

Comment:
not entirely sure if that's what we want (i.e. whether we want to DRY and automate this), but you could grab the version from the top-level `mix.exs` file and use functions around https://hexdocs.pm/elixir/Version.html to pull major.minor out of that.

But maybe this is not such a good idea to couple the two so strongly :thinking: ?...

Reply:
The idea is to couple this API reported version with EIP712 domain - which means OMG network version. This is separated from mix / application version.